apps/website/src/data/found-animal.ts

```diff
@@ -242,6 +242,150 @@ const data: FoundAnimalFlow = {
         ],
       },
     },
+    {
+      name: "Squirrel",
+      flow: {
+        prompt: [
+          "Does any of the following apply to the squirrel?",
+          "- It is bleeding, has an open wound, or has a broken bone",
+          "- It has been in a cat's or dog's mouth",
+          "- It is covered in fly eggs (looks like small grains of rice)",
+          "- It is cold, wet, or crying nonstop",
+        ],
+        options: [
+          {
+            name: "Yes",
+            flow: {
+              prompt: macros.general.rehab("the squirrel"),
+            },
+          },
+          {
+            name: "No",
+            flow: {
+              prompt: [
+                "If the squirrel is uninjured the next step is to identify its age to determine if intervention is needed",
+                "Does any of the following apply to the squirrel",
+                "- have a fluffed out tail",
+                "- a body longer than 6 inches (not including the tail)",
+                "- approach humans or pets",
+              ],
+              options: [
+                {
+                  name: "Yes",
+                  flow: {
+                    prompt: [
+                      "This is a juvenile squirrel, you do not need to intervene.",
+                      ...macros.general.leave("the squirrel"),
+                    ],
+                  },
+                },
+                {
+                  name: "No",
+                  flow: {
+                    prompt: [
+                      "This is an infant squirrel, You will need to guide the baby back to its mother",
+                      "Place uncooked rice or bird seed in a sock and warm in the microwave for 20-30 seconds, Wrap the sock in a soft towel, and place it with the baby squirrel in an open container (e.g. a box)",
+                      "Do NOT give the baby food or water",
```

Comment:
```suggestion
                      "Do not attempt to feed or give water to the squirrel, as this may lead to it imprinting on humans and becoming dependent on them.",
```